src/tags_int.cpp

```diff
@@ -106,7 +106,8 @@ namespace Exiv2 {
         { nikonWtId,       "Makernote", "NikonWt",      Nikon3MakerNote::tagListWt     },
         { nikonIiId,       "Makernote", "NikonIi",      Nikon3MakerNote::tagListIi     },
         { nikonAfId,       "Makernote", "NikonAf",      Nikon3MakerNote::tagListAf     },
-        { nikonAf2Id,      "Makernote", "NikonAf2",     Nikon3MakerNote::tagListAf2    },
+        { nikonAf21Id,     "Makernote", "NikonAf2",     Nikon3MakerNote::tagListAf21   },
```

Comment:
[Suggestion] For consistency I'd suggest to call list & enumeration element `*Af2` instead of `*Af21`.

Reply:
You're right about this.  We have two things which are very similar `nikonAf2ID` (AutoFocus version 1.00) and `nikonAf22Id` (AutoFocus version 1.01).

Reply:
Dan: As always you are very sharp eyed.  When I developed the code, I changed nikonAf2Id to nikonAf21Id and cut'n'pasted/morphed nikonAf21Id into nikonAf22Id.  Later I discovered this broke the test suite, because there's a test that uses `Exif.NikonAf2.something`.  I hacked that table to pass the test suite.  I've restored nikonAf2Id.

It's really hard to understand the tiff parsing code.  It's very abstract.  I've been trying for 2 days to fix #582 and haven't got it work yet.  I'll defer #582 until v0.27.3.